docs/src/sources-cog-files.md

```diff
@@ -0,0 +1,82 @@
+# Cloud Optimized GeoTIFF File Sources
+
+Martin can serve local [COG(Cloud Optimized GeoTIFF)](https://cogeo.org/) files. For cog on remote like S3 and other improvements, you could track them on [issue 875](https://github.com/maplibre/martin/issues/875), we are working on and welcome any assistance.
```

Comment:
Could you add a short warning/info section which makes clear that COGs are images and not a vector source? ^^
Also, what do we need to change here https://maplibre.org/martin/using.html

Reply:
I guess no need to change the Usage and Endpoint APi?

Reply:
Tif or tiff? Seems to have both, but should probably use one?

Reply:
@nyurik Which one do you perfer? 
[Tiff 6.0 spec](https%3A%2F%2Fwww.itu.int%2Fitudoc%2Fitu-t%2Fcom16%2Ftiff-fx%2Fdocs%2Ftiff6.pdf), [geotiff doc](https://docs.ogc.org/is/19-008r4/19-008r4.html) and [cog doc](https://docs.ogc.org/is/21-026/21-026.html) all use `tiff` as part of their doc name. So we use tiff?

Reply:
The confusing part is generally the file extension is `tif` and the doc names are `tiff`.

Reply:
And another to discuss, should we support `*.tiff` and `*.tif` both? Seems they are both right. But I rarely see a `.tiff` file.

Reply:
And if we do support both, do them next PR.

Reply:
I think we should auto-discover both